larray/core/axis.py

```diff
@@ -1194,6 +1195,34 @@ def align(self, other, join='outer'):
                 other = Axis(other)
             return other
 
+    def to_hdf(self, filepath, key, **kwargs):
+        """
+        Writes axis to a HDF file.
+
+        A HDF file can contain multiple axes.
+        The 'key' parameter is a unique identifier for the axis.
+
+        Parameters
+        ----------
+        filepath : str
+            Path where the hdf file has to be written.
+        key : str or Group
+            Name of the axis within the HDF file.
+        **kargs
+
+        Examples
+        --------
+        >>> a = Axis("a=a0..a2")
+        >>> a.to_hdf('test.h5', 'a')       # doctest: +SKIP
+
+        Save axis in a specific HDF group
+
+        >>> a.to_hdf('test.h5', 'axes/a')  # doctest: +SKIP
+        """
+        key = _translate_key_hdf(key)
+        s = pd.Series(data=self.labels, name=self.name)
```

Comment:
Can't we go via LArray instead of via Series?

Reply:
What would be the added value of using LArray since LArray use pandas internally?

Reply:
1. less reliance on pandas, if we ever want to provide our own serializers
2. different output format. This is both a good thing (arrays and axes use the same format) and a bad thing (the format is more verbose).

Reply:
1. Considering the time to develop our own serializers divided by the maximum number of users we can expect, I'm not sure it will ever worth it. pandas makes some sanity checks that we don't need to implement ourselves.   
2. While I'm developing the code to import/export Axis and Group objects, I keep in mind that some day we will also include ``metadata`` to sessions (i.e. objects of basic types like string, int, float, boolean, list and dictionary). So, I'm not sure it is a good idea to go via LArray.

Why not using ``pytables`` directly instead of pandas for non-LArray objects?

Reply:
1. In general, it is a good idea to use the higher level interface than the lower level one to do the same thing, because it simplifies maintenance. In some cases, the performance difference justifies calling the lower level API directly. Here I consider pandas to be the lower level API. That being said, in this case the different format is more important than the ease of maintenance so going via pandas is probably the better option.
2. you are right but I don't see how it relates to my previous comment.

> Why not using pytables directly instead of pandas for non-LArray objects?

We could but pandas is so prevalent, that we need to be able to read (and possibly write) the pandas dialect of hdf, which has a very specific and a bit odd way to store objects.

Reply:
So I will continue to rely on pandas. For metadata, there is a solution: http://pandas-docs.github.io/pandas-docs-travis/cookbook.html#hdfstore (see also https://stackoverflow.com/questions/29129095/save-additional-attributes-in-pandas-dataframe/29130146#29130146)